tests/pipeline/nodes/augment/undistort/camera_calibration_coeffs.yml

```diff
@@ -0,0 +1,7 @@
+camera_matrix:
+- [1068.4989359271717, 0.0, 1000.4735395825165]
+- [0.0, 1087.775997855007, 530.5860397938737]
+- [0.0, 0.0, 1.0]
+distortion_coeffs:
+- [-0.3596139644552646, 0.13579142330603097, -0.0011853301906309458, -0.00019870976862568617,
+  -0.02416638699314911]
```

Comment:
Same as above, can this file be moved to `tests/data/camera_calibration/` instead?